src/types.ts

```diff
@@ -261,6 +262,7 @@ export type RenderType =
   | 'content-switcher'
   | 'date'
   | 'datetime'
+  | 'time'
```

Comment:
Here you're introducing a new rendering that is supported through the "date" rendering. With the introduction of the `datePickerFormat` prop, we only need one rendering "date". Meaning we should consider removing or deprecating the "datetime" rendering.

Reply:
Adding `time` was a mistake so I removed it. Leaving `datetime` as it is for now as it has being referenced in a couple of places.

Reply:
I think we need to update existing references to ensure that we have a single source of truth. Eg. We should use `datePickerFormat` [here](https://github.com/openmrs/openmrs-form-engine-lib/blob/main/src/submission-handlers/obsHandler.ts#L163-L165) to determine the date-picker type.